.github/workflows/release_pypi.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-20.04, macos-14, macos-15, windows-latest]
+        os: [ubuntu-latest, macos-14, macos-15, windows-latest]
```

Comment:
`ubuntu-20.04` is quite old ... I hope we can run that job with `ubuntu-latest`?!

Reply:
It was added since the beginning, I suppose, didn't find answer on why this exact version: https://github.com/dottxt-ai/outlines-core/commit/909c7f2280ea331afdf91ee46ba0699bb6d00b8d

So, hopefully, yes.